pkg/network/setup/netstat.go

```diff
@@ -213,18 +213,8 @@ func ifacesStatusFromDomainInterfaces(domainSpecIfaces []api.Interface) []v1.Vir
 	var vmiStatusIfaces []v1.VirtualMachineInstanceNetworkInterface
 
 	for _, domainSpecIface := range domainSpecIfaces {
-		var ifaceAlias string
```

Comment:
Out of curiosity, why was this removed? Can this have any negative impact on non-cloud-hypervisor platforms?

Reply:
I guess I am curious about what "Hermes" is. Do you happen to know?

Reply:
The code that has been removed was actually added as a workaround because Libvirt's cloud-hypervisor driver was not preserving aliases. Now that this problem in libvirt has been fixed, I am removing that code.

Also, Hermes was the codename of the project under which we started working to generalize the virtualization stack.

Reply:
Thanks for the details, Harshit! Makes sense